### ISSUE TITLE
New Post title textarea height changes when focused on element (desktop)

### ISSUE THREAD

**Issue by Coteh** [forem/forem#10261]
**Describe the bug**

Whenever I create a post on DEV, after focusing on the textarea for the title, its height changes to a smaller value that makes it hard to read. Screenshots attached below. I work around this for the time being by adjusting the height value myself using the inspect window. I turned off uBlock Origin to make sure that wasn't doing anything, but the problem still persists. I will update this post once I try it with all extensions disabled.

**To Reproduce**

1. When logged in, go to "New Post"
2. Click on post title textarea to bring it into focus

The textarea will continue to have this new height value even after clicking off the textarea.

**Expected behavior**

Textarea height should remain the same.

**Screenshots**

<img width="1680" alt="Screen Shot 2020-09-09 at 11 43 03 AM" src="https://user-images.githubusercontent.com/3276350/92622263-dae7e900-f292-11ea-93a6-70a076498d08.png">

<img width="1680" alt="Screen Shot 2020-09-09 at 11 43 11 AM" src="https://user-images.githubusercontent.com/3276350/92622278-de7b7000-f292-11ea-876e-9fb694889f27.png">

As you can see here, the height changes from 60px to 27px once in focus.

**Desktop (please complete the following information):**

- OS, version: macOS Mojave 10.14.6
- Browser, version: Firefox 80.0.1 (64-bit)

**Comment by rhymes** [comment]
Hi @Coteh, thanks for the report! This is likely the same issue experienced by this reporter here https://github.com/forem/forem/issues/10114

Can you confirm? Could you add your report to the page if so?

Interesting because the combination of browser and operating system is totally different then theirs

**Comment by Coteh** [comment]
@rhymes Yup looks like the same one. Thanks for mentioning this, added a comment to the other issue.

**Comment by rhymes** [comment]
Thank you @Coteh!
